tests/baselines/reference/constEnumToStringWithComments.js

```diff
@@ -25,15 +25,15 @@ let c1 = Foo["C"].toString();
 
 
 //// [constEnumToStringWithComments.js]
-var x0 = 100 /* Foo.X */.toString();
-var x1 = 100 /* Foo["X"] */.toString();
+var x0 = (100 /* Foo.X */).toString();
+var x1 = (100 /* Foo["X"] */).toString();
 var y0 = 0.5 /* Foo.Y */.toString();
 var y1 = 0.5 /* Foo["Y"] */.toString();
-var z0 = 2 /* Foo.Z */.toString();
-var z1 = 2 /* Foo["Z"] */.toString();
-var a0 = -1 /* Foo.A */.toString();
-var a1 = -1 /* Foo["A"] */.toString();
-var b0 = -1.5 /* Foo.B */.toString();
-var b1 = -1.5 /* Foo["B"] */.toString();
-var c0 = -1 /* Foo.C */.toString();
-var c1 = -1 /* Foo["C"] */.toString();
+var z0 = (2 /* Foo.Z */).toString();

```

Comment:
`2.toString()` is invalid but `2/**/.toString` is so those parenthesis could also be avoided here, OTOH I'm not really sure if it's worth avoiding them here 🤷‍♂️